frappe/automation/doctype/auto_repeat/auto_repeat.py

```diff
@@ -8,11 +8,11 @@
 import frappe
 from frappe import _
 from frappe.automation.doctype.assignment_rule.assignment_rule import get_repeated
+from frappe.comm.doctype.communication.email import make
```

Comment:
```suggestion
from frappe.communications.doctype.communication.email import make
```

Let's go with full name for better readability 😅

Reply:
Addressed in [force-push](https://github.com/frappe/frappe/compare/ac242bc6c0c766467dd79629af24797c0bb46cc5..989d70f975d6faa0e6ddbe6618cb2d116173dd85)